lib/slide_hub/io.rb

```diff
@@ -0,0 +1,9 @@
+class IO
+  def head(n)
```

Comment:
__[Reek]__

[UncommunicativeParameterName] IO#head has the parameter name 'n'


<!-- Comment by [SideCI](https://sideci.com) -->